content/sensu-go/5.5/installation/recommended-hardware.md

```diff
@@ -11,67 +11,99 @@ menu:
     parent: installation
 ---
 
+- [Sensu backend requirements](#sensu-backend)
+- [Sensu agent requirements](#sensu-agent)
+- [Networking recommendations](#networking-recommendations)
+- [Cloud recommendations](#cloud-recommendations)
+
+## Sensu backend
+
 ### Backend minimum requirements
 
+The following configuration is the minimum required to run the Sensu backend, however it is insufficient for production use.
+See the [recommended configuration](#backend-recommended-configuration) for production recommendations.
+
 * 64-bit Intel or AMD CPU
 * 4 GB RAM
 * 4 GB free disk space
 * 10 mbps network link
 
 ### Backend recommended configuration
 
+The following configuration is recommended as a baseline for production use to ensure a good user and operator
+experience. Using additional
+resources (even over-provisioning) further improves stability and
+scalability.
+
 * 64 bit 4-core Intel or AMD CPU
 * 8 GB RAM
-* SSD (NVMe or SCSI)
+* SSD (NVMe or SATA3)
 * Gigabit ethernet
 
-The Sensu backend is typically CPU and storage intensive. Its use of these
-resources scales linearly with the total number of checks that all agents
-connecting to the backend are executing.
-
-Sensu backend is a massively parallel application that can scale to any number
-of CPU cores. Provision roughly 1 CPU core for every 50 checks per second,
-including keepalives.
+The Sensu backend is typically CPU and storage intensive. In general, its use of
+these resources scales linearly with the total number of
+checks executed by all Sensu agents connecting to the backend.
 
+The Sensu backend is a massively parallel application that can scale to
+any number of CPU cores. Provision roughly 1 CPU core for every 50
+checks per second (including agent keepalives).
 Most installations are fine with 4 CPU cores, but larger installations
-may find that 8 CPU cores are necessary.
+may find that additional CPU cores (8+) are necessary.
 
-Each check that is executed results in writes occurring in the backend.
-When provisioning storage hardware, a good rule of thumb is to have twice as
-many iops as you expect to have checks executing per second. Don't forget to
-include keepalives in this calculation; each agent executes a keepalive
-every 20 seconds. So in a cluster of 100 agents, you can expect those agents
-to consume 10 write iops for keepalives.
+Every executed Sensu check results in storage writes. When
+provisioning storage, a good rule of thumb is to have twice as many
```

Comment:
```suggestion
provisioning storage, a good guideline is to have twice as many
```

Reply:
Fixed in https://github.com/sensu/sensu-docs/pull/1371/commits/fac248bb22988a0f5347388f32fac7a6135a701b